integration-libs/cdc/user-profile/update-email/cdc-update-email-component.service.ts

```diff
@@ -13,11 +14,17 @@ import {
   GlobalMessageType,
   RoutingService,
 } from '@spartacus/core';
-import { UpdateEmailComponentService } from '@spartacus/user/profile/components';
+import {
+  USE_MY_ACCOUNT_V2_EMAIL,
+  UpdateEmailComponentService,
+} from '@spartacus/user/profile/components';
 import { UserEmailFacade } from '@spartacus/user/profile/root';
+import { Subject } from 'rxjs';
 
 @Injectable()
 export class CDCUpdateEmailComponentService extends UpdateEmailComponentService {
+  updateSucceed$ = new Subject();
+  enableMyAccountV2 = inject(USE_MY_ACCOUNT_V2_EMAIL);
```

Comment:
[MAJOR]
In other {MAJOR} comment I asked if we could possibly not create MyAccountV2-flavored component-services, but just make small changes in the original ones. Then the consequence would be that we would likely not need v2-flags conditions in the CDC-flavored component-service.

Of course it's still open for discussion, whether we will leave separate MyAccountV2-flavored component-services. But let me leave this comment here, just as a reminder to followup on the topic CDC component-services after we make the decision about the MyAccountV2 component-services.

Reply:
Since CDCUpdatexxxxComponentService will overwrite V1XxxxService, and there are some functions redefined in CDCUpdatexxxxComponentService. In this condition, I'm afraid we can’t avoid to touch cdc files.

Reply:
allo reverted